src/main/java/internet/market/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package internet.market.service.impl;
+
+import internet.market.dao.UserDao;
+import internet.market.lib.Inject;
+import internet.market.lib.Service;
+import internet.market.model.User;
+import internet.market.service.ShoppingCartService;
+import internet.market.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Inject
+    private UserDao userDao;
+
+    @Inject
+    private ShoppingCartService shoppingCartService;
```

Comment:
do we use this field anywhere in the class?

Reply:
done